Formats/Pauper/metas.json

```diff
@@ -1,15 +1,25 @@
-
 {
-   "Metas":
+   Metas:
    [
-        { "StartDate": "2019-05-20", "Name": "BlueMonday" }, 
-        {"StartDate": "2019-06-27", "Name" : "PauperMaster"},
-        {"StartDate": "2019-06-14" , "Name":"ModernHorizons1"}, 
-        {"StartDate":"2019-10-21" , "Name":"PostModernHorizon1"},
-        {"StartDate": "2020-07-13" , "Name":"PostSanctuary"}, 
-        {"StartDate": "2020-11-20" , "Name":"CommanderLegend"}, 
-        {"StartDate": "2021-01-14" , "Name":"PostFallFromFlavor"}, 
-        {"StartDate": "2021-06-18" , "Name":"ModernHorizons2"}, 
-        {"StartDate": "2021-09-08" , "Name":"PostChatterStorm"}
+      { "StartDate": "2019-05-20", "Name": "PostBlueMonday" }, 
+      { "StartDate": "2019-06-14", "Name": "PostModernHorizons"},
+      { "StartDate": "2019-06-27", "Name": "PostPauperMaster"},
```

Comment:
I'm assuming this is the one that was previously listed as "PostPauperMaster" since that's the date of the article.
https://magic.wizards.com/en/articles/archive/news/pauper-comes-paper-2019-06-27

Fixed.